### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Page-**
Update @balena/abstract-sql-compiler from 9.0.3 to 9.0.4 Update @balena/odata-to-abstract-sql from 5.9.6 to 6.2.3

Change-type: patch

### CODE REVIEW

**Review by otaviojacobi — CHANGES_REQUESTED** [review]
@Page- I think you have lint & lint-fix reversed

File: package.json [hunk 1]
```diff
@@ -8,35 +8,36 @@
     "pretest": "npm run lint && npm run prepare",
     "test": "mocha",
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\" && tsc --project ./tsconfig.build.json",
-    "lint": "balena-lint --typescript --fix src test && tsc --noEmit"
+    "lint-fix": "balena-lint src test && tsc --noEmit",
+    "lint": "balena-lint --fix src test && tsc --noEmit"
```
Comment (otaviojacobi) [inline package.json:12]: Here lint-fix should have --fix & lint should just do the check

**Comment by otaviojacobi** [comment]
@Page-  I think you migh have forgotten to push

**Comment by Page-** [comment]
Apparently so, fixed @otaviojacobi

**Review by otaviojacobi — APPROVED** [review]

**Pull request merged**